modules/overrides/overrides.go

```diff
@@ -214,43 +223,57 @@ func (o *Overrides) IngestionRateStrategy() string {
 // MaxLocalTracesPerUser returns the maximum number of traces a user is allowed to store
 // in a single ingester.
 func (o *Overrides) MaxLocalTracesPerUser(userID string) int {
-	return o.getOverridesForUser(userID).MaxLocalTracesPerUser
+	x := o.getOverridesForUser(userID).MaxLocalTracesPerUser
+	metricOverridesLimits.WithLabelValues("max_local_traces_per_user", userID).Set(float64(x))
```

Comment:
I think it would be better if we recorded these values when they are periodically loaded from the overrides file (I think in the `loadPerTenantOverrides` function above)  since that's the only time those values can change, rather than on each retrieval as its done currently.

Reply:
Here is how cortex is doing it:
https://github.com/cortexproject/cortex/blob/master/pkg/util/validation/exporter.go

Reply:
Updating the metrics on file load makes sense to me.  Do we want to introduce a channel for this to copy cortext as well?

Reply:
> Do we want to introduce a channel for this to copy context as well?

Unsure what you're asking, but I would like to use the exporter pattern like the linked cortex code.